### PR TITLE
Fix existence test failures due to entity value varies during upgrade

### PR DESCRIPTION
**Pull request by jyejare**
Fixes issue #54

### CODE REVIEW

**Comment by renzon** [comment]
Once would be harder to explain my idea than to implement, I sent @jyejare a PR: https://github.com/jyejare/satellite6-upgrade/pull/1. The idea is using a dict for now. And once more sat versiosn land, we can make more diff dict and make a recursive lookup based on from and to version. But once at this time the code is dealing with only 6.1 to 6.2, I kept it simple.

**Comment by jyejare** [comment]
@renzon , I saw your suggested code and have some comments here : https://github.com/jyejare/satellite6-upgrade/pull/1

**Comment by renzon** [comment]
Checked and fixed your [comments there](https://github.com/jyejare/satellite6-upgrade/pull/1/files). I implemented tests regarding name changes for:
6.1 -> 6.2
6.1 -> 6.3
6.2 -> 6.3

So even if you prefer you solution, can you add those tests to it?

**Comment by jyejare** [comment]
@renzon , the code patch you sent to me look closed. Do you have implemented it somewhere else.

**Comment by renzon** [comment]
I sent another PR, the one you commented you prefer your version.

**Comment by jyejare** [comment]
@renzon , I dont see any comments on review. Are you talking about the closed PR's by you.

**Comment by jyejare** [comment]
@SatelliteQE/quality-engineers , Need reviewers please visit.

**Comment by renzon** [comment]
No, there 2 reviews here: https://github.com/SatelliteQE/satellite6-upgrade/pull/62/files

**Comment by renzon** [comment]
Pleaso also wright some test cases for assert_varient so we can grasp its use ;)

File: upgrade_tests/helpers/variants.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+# }
+#
+# Note: The variants should be listed from 6.1 onwards
+entity_varients = {
+    'filter': [
+        ['lookupkey', 'variablelookupkey'],
+        ['(miscellaneous)', 'foremanopenscap::arfreport'],
+        ['organization', 'katello::subscription'],
+        ['configtemplate', 'provisioningtemplate'],
+        ['view_templates, create_templates, edit_templates, '
```
Comment (jyejare) [inline upgrade_tests/helpers/variants.py]: Yes :)

**Pull request merged**